src/theme.ts

```diff
@@ -249,60 +258,60 @@ export const theme: Theme = {
     button1: {
       ..._theme.typography.button1,
       // original Figma fontSize: 30px
-      fontSize: pxToRem(24),
-      [sm]: {fontSize: pxToRem(24)},
-      [md]: {fontSize: pxToRem(24)},
-      [lg]: {fontSize: pxToRem(24)},
+      fontSize: pxToRem(20),
+      [sm]: {fontSize: pxToRem(20)},
+      [md]: {fontSize: pxToRem(18)},
+      [lg]: {fontSize: pxToRem(22)},
       [xl]: {fontSize: pxToRem(30)},
       lineHeight: 1.5,
     },
     button2: {
       ..._theme.typography.button2,
       // original Figma fontSize: 20px
-      fontSize: pxToRem(14),
-      [sm]: {fontSize: pxToRem(16)},
-      [md]: {fontSize: pxToRem(16)},
-      [lg]: {fontSize: pxToRem(18)},
+      fontSize: pxToRem(12),
+      [sm]: {fontSize: pxToRem(12)},
+      [md]: {fontSize: pxToRem(13)},
+      [lg]: {fontSize: pxToRem(16)},
       [xl]: {fontSize: pxToRem(20)},
       lineHeight: 1.5,
     },
     button3: {
       ..._theme.typography.button3,
       // original Figma fontSize: 14px
-      fontSize: pxToRem(10),
-      [sm]: {fontSize: pxToRem(10)},
-      [md]: {fontSize: pxToRem(10)},
-      [lg]: {fontSize: pxToRem(12)},
+      fontSize: pxToRem(11),
+      [sm]: {fontSize: pxToRem(11)},
+      [md]: {fontSize: pxToRem(9)},
```

Comment:
zvlastne vidiet 11/11/9 ale asi ok 😄

Reply:
cele tie font size su take divne, tiez ma boleli prsty ked som to pisal